src/commons_abstract_fevalues.jl

```diff
@@ -1,4 +1,4 @@
-# Common methods for both FECellValues and FEBoundaryValues
+# Common methods for both FECellScalarValues and FEBoundaryValues
```

Comment:
see above, isnt this commone methods for all values now? eg `Common methods for AbstractFEValues`